onnxruntime/core/common/helper.cc

```diff
@@ -26,8 +26,10 @@ std::string ToMBString(const std::wstring& s) {
   const int len = WideCharToMultiByte(CP_ACP, 0, s.data(), src_len, nullptr, 0, nullptr, nullptr);
   assert(len > 0);
   std::string ret(static_cast<size_t>(len) - 1, '\0');
+#pragma warning(disable: 4189)
   const int r = WideCharToMultiByte(CP_ACP, 0, s.data(), src_len, (char*)ret.data(), len, nullptr, nullptr);
   assert(len == r);
+#pragma warning(default: 4189)
```

Comment:
Because r is not referenced in release mode?

Reply:
Correct.